#211 - about company page, update main page

AleksandraKozubal opened the pull request:
Created about page, but also added simple bubbles animation, change button placement and hover buttons style in nav, added binary img in footer, switched tech images to similarly sized ones.
![image](https://github.com/blumilksoftware/website/assets/104600942/89852bd7-c55b-4d9a-b842-80db5512d036)


It should close #211

EwelinaSkrzypacz reviewed (commented):
1. I don't know why, but on my laptop, the font is different form that in design


My laptop:

![image](https://github.com/blumilksoftware/website/assets/56546832/6be25ea4-3010-42b5-a714-53575b9840d9)


Design:

![image](https://github.com/blumilksoftware/website/assets/56546832/949cf555-14b7-4d67-be8d-59c5fe0744c5)


2. When I hover on images, they overlap a bit - please see the video below

https://github.com/blumilksoftware/website/assets/56546832/149b6b36-2d7c-4b7d-9093-05a1078b5560


3. On mobile - too much space above the image

![image](https://github.com/blumilksoftware/website/assets/56546832/398b10a2-dc11-45a4-be37-c8d2e4cfac95)

4. On mobile I can scroll left-right, but I shouldn't.

AleksandraKozubal commented:
> 1. I don't know why, but on my laptop, the font is different form that in design
> 
>  2. When I hover on images, they overlap a bit - please see the video below
> 
>  3. On mobile - too much space above the image
> 
>  4. On mobile I can scroll left-right, but I shouldn't.

1. changed
2. We decided to leave it for now, coz it looks ok on my laptop, tested on firefox and edge
3. removed
4. fixed

EwelinaSkrzypacz reviewed (dismissed):
I created issue for this overlapping images -> https://github.com/blumilksoftware/website/issues/248

But we will leave for now, so :+1:

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+                        </div>
+                        <div class="h-36 2xl:h-52 3xl:h-72 w-full place-items-center md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                             data-taos-offset="400">
+                            <img src="{{ asset('images/photos/birthday.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-105" alt="placeholder"/>
```
```suggestion
                            <img src="{{ asset('images/photos/birthday.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-105" alt="placeholder"/>
```

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+                                </div>
+                                <div class="h-16 2xl:size-24 3xl:size-32 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/plane.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
```suggestion
                                    <img src="{{ asset('images/photos/plane.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+                            <div class="grid grid-rows-2 justify-self-center gap-2 2xl:gap-6">
+                                <div class="h-16 2xl:size-24 3xl:size-32 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/meetup.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
```suggestion
                                    <img src="{{ asset('images/photos/meetup.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+                                </div>
+                                <div class="h-16 2xl:h-24 3xl:h-32 w-40 xl:w-52 2xl:w-60 3xl:w-80 delay-300 duration-1000 taos:scale-75 taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/energylandia.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
```suggestion
                                    <img src="{{ asset('images/photos/energylandia.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+                            <div class="grid col-span-2 grid-rows-2 place-content-evenly place-items-center gap-2 2xl:gap-6">
+                                <div class="h-16 2xl:h-24 3xl:h-32 w-40 xl:w-52 2xl:w-60 3xl:w-80 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                                     data-taos-offset="400">
+                                    <img src="{{ asset('images/photos/rzeka.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```
```suggestion
                                    <img src="{{ asset('images/photos/rzeka.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl transform transition hover:scale-110" alt="placeholder"/>
```

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+            <div class="flex xl:hidden">
+                <div class="size-full 2xl:size-52 3xl:size-72 md:delay-300 md:duration-1000 md:taos:scale-75 md:taos:opacity-0"
+                     data-taos-offset="400">
+                    <img src="{{ asset('images/photos/lasertag.jpg') }}" class="h-full w-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl" alt="placeholder"/>
```
```suggestion
                    <img src="{{ asset('images/photos/lasertag.jpg') }}" class="size-full object-cover rounded-xl 2xl:rounded-2xl 3xl:rounded-3xl" alt="placeholder"/>
```

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+                </div>
+            </div>
+            <div class="hidden xl:flex basis-2/3 gap-2 2xl:gap-6 justify-center">
+                <div class="grid grid-cols-1 place-items-center gap-2 2xl:gap-4">
+                    <div class="grid grid-rows-2 place-items-center gap-2 2xl:gap-6">
+                        <div class="grid grid-cols-3 place-content-evenly gap-2 2xl:gap-6">
+                            <div class="grid col-span-2 grid-rows-2 place-content-evenly place-items-center gap-2 2xl:gap-6">
```
So may `grid` class I see here. Can this be simplified?

AleksandraKozubal replied:
yes, changed that

dawidrudnik commented on resources/views/about.blade.php:
```diff
@@ -1,5 +1,172 @@
+    <section class="bg-gradient-to-b from-gray-light to-white">
+        <x-waves.wave-2.top></x-waves.wave-2.top>
+        <div class="relative flex flex-col xl:flex-row text-center md:text-left px-[10%] md:px-[20%] gap-x-16 py-8 xl:py-28 z-20">
+            <div class="flex flex-col basis-1/3">
```
`basis-1/3` class can be problematic in mobile version.

AleksandraKozubal replied:
switched to grid, right now grid cols 3 is used only on 2xl screens so it shouldn't be a problem

AleksandraKozubal commented:
In the last commit I changed a few things after @bl4cksugar review.

EwelinaSkrzypacz reviewed (commented):
1. I like the idea of language in dropdown, but this language list is too narrow and it's rounded, but I think it should be not rounded.

![image](https://github.com/blumilksoftware/website/assets/56546832/5ed6c172-7c44-4864-82d7-6453a3d79998)

2. Really nice that we removed all animations, because now its a lot better. But when I refresh page I see that logo and CTA button on navigation are moving - can we fix this somehow?

https://github.com/blumilksoftware/website/assets/56546832/e40ada30-9aac-4bc9-89b8-785802c13ee2

3. I don't know why some images are not zoom a bit, when I hover mouse cursor
(and funfact - somehow now photos are not overlapping anymore)


https://github.com/blumilksoftware/website/assets/56546832/67eff131-ed9e-49b0-ad38-55404a0acfe5

AleksandraKozubal commented:
> 1. I like the idea of language in dropdown, but this language list is too narrow and it's rounded, but I think it should be not rounded.
> 
>  2. Really nice that we removed all animations, because now its a lot better. But when I refresh page I see that logo and CTA button on navigation are moving - can we fix this somehow?
> 
> 3. I don't know why some images are not zoom a bit, when I hover mouse cursor
>        (and funfact - somehow now photos are not overlapping anymore)

1. changed, it is square as dropdown toggle
2. It should be better right now
3. fixed

EwelinaSkrzypacz reviewed (commented):
Line height for bold text is different than line height fot normal text

![image](https://github.com/blumilksoftware/website/assets/56546832/c493491b-280a-4de6-aa8e-f8fc551921e7)

AleksandraKozubal commented:
> Line height for bold text is different than line height fot normal text
> 
> ![image](https://private-user-images.githubusercontent.com/56546832/334363191-c493491b-280a-4de6-aa8e-f8fc551921e7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTY4OTEyNDQsIm5iZiI6MTcxNjg5MDk0NCwicGF0aCI6Ii81NjU0NjgzMi8zMzQzNjMxOTEtYzQ5MzQ5MWItMjgwYS00ZGU2LWFhOGUtZjhmYzU1MTkyMWU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTI4VDEwMDkwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU0ZDE3NzRmYTU3MGQ1NzVkYTczNGZlNjNlN2NjMGQ1NDY5Zjk3ZjNkZjIyY2UwYTBkNjRjMDI4Yjg0MTE2NmQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.da3U4Dn4AWFsxlZZu8gR4qYOBrsPALP4LFmgSvKSX0w)

fixed
